rest-api-spec/src/main/resources/rest-api-spec/api/query_rule.delete.json

```diff
@@ -0,0 +1,35 @@
+{
+  "query_rule.delete": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/delete-query-rule.html",
+      "description": "Deletes an individual query rule within a ruleset."
+    },
+    "stability": "experimental",
```

Comment:
The `stability` will be updated in a followup PR that marks the entire query rules feature as stable.